### PR TITLE
Hunt groups rel user filters

### PR DESCRIPTION
**Pull request by mmadariaga**
Reviewed HuntGroupRelUser logic:
 - Made priority and timeout required for non ringAll hunt groups
 - Fixed klear error on user -> huntGroupRelUser -> delete dialog
 - Do not allow to edit/create huntGroupRelUser through user screen anymore. It's still possible to check the list and remove items

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

**Comment by Kaian** [comment]
We'll wait for #901 to merge this and compete @todo sections. 

The rest seems good to me.

**Comment by Kaian** [comment]
Unblocked

File: library/Ivoz/Provider/Domain/Model/HuntGroupsRelUser/HuntGroupsRelUser.php [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
 class HuntGroupsRelUser extends HuntGroupsRelUserAbstract implements HuntGroupsRelUserInterface
 {
     use HuntGroupsRelUserTrait;
 
+    protected function sanitizeValues()
+    {
+        $huntGroup = $this->getHuntGroup();
+
+        /** @todo make ringAll HuntGroup constant */
+        if ($huntGroup->getStrategy() === 'ringAll') {
```
Comment (Kaian) [inline library/Ivoz/Provider/Domain/Model/HuntGroupsRelUser/HuntGroupsRelUser.php]: Constant here

Comment (mmadariaga) [reply library/Ivoz/Provider/Domain/Model/HuntGroupsRelUser/HuntGroupsRelUser.php]: Removed :)

**Pull request merged**